### PR TITLE
Add header / footer component

### PR DESCRIPTION
**Pull request by ben181231**
connect #5

### CODE REVIEW

File: components/Footer/Footer.scss [hunk 1]
```diff
@@ -0,0 +1,92 @@
+@import '../variables.scss';
+
+.Footer{
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #000000;
+  color: #ffffff;
```
Comment (rickmak) [inline components/Footer/Footer.scss]: Use color from `variables.scss`?

**Comment by rickmak** [comment]
Do you think the alignment on footer is a bit off top left? (Not saying your implementation, but the design)

File: components/Footer/Footer.scss (hunk 1 above)
Comment (ben181231) [inline components/Footer/Footer.scss]: Okay.

**Comment by ben181231** [comment]
@rickmak 
The left side aligns to the header and contents. But the right side is a little bit off (I add some margin according to the design). 
Do you think I need to remove the margin ?

**Comment by ben181231** [comment]
@rickmak Updated.

**Pull request merged**